### PR TITLE
Put how-to on the homepage

### PR DESCRIPTION
**Pull request by lemairecarl**
* Put the how-to on the homepage
* Add a "Recent posts" page

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,6 +1,61 @@
...
+hidden:     true  # Excluded from the menu
 ---
+
+<style>
+    li {
+        margin-bottom: 0.7rem; /* This will make the lists easier to read. */
+    }
+</style>
+
+**Welcome!** Through this website, we will help each other stay up to date with new advances in our ever changing field. Our paper summaries will also help new members of VITALab to acquire essential knowledge about the state of the art. On this site, you'll find all the essential papers the VITALab members should be aware of.
```
Comment (jodoin) [inline index.md]: The goal of this website is for the VITALab member to stay up to date with new scientific publications.  This site also helps new members acquire knowledge on fundamental methods.

File: index.md [hunk 2]
```diff
@@ -1,6 +1,61 @@
...
+
+## How to add a review
+
+The process for adding reviews is _git-centric_. Basically, **you just need to add a file to the repo and make a pull request**. Let's go into the details :
+
+1.  Clone [the `vitalab.github.io` repo](https://github.com/vitalab/vitalab.github.io) on your computer.
+2.  Determine the category in which you will add your post. Categories are managed using folders :
```
Comment (jodoin) [inline index.md]: in which you want to add your post.

File: index.md [hunk 3]
```diff
@@ -1,6 +1,61 @@
...
+
+## How to add a review
+
+The process for adding reviews is _git-centric_. Basically, **you just need to add a file to the repo and make a pull request**. Let's go into the details :
+
+1.  Clone [the `vitalab.github.io` repo](https://github.com/vitalab/vitalab.github.io) on your computer.
+2.  Determine the category in which you will add your post. Categories are managed using folders :  
+    
+    ~~~
+    _posts/           # A post added here will have no category
```
Comment (jodoin) [inline index.md]: will not be associated to a category.

File: index.md [hunk 4]
```diff
@@ -1,6 +1,61 @@
...
+    course/
+        _posts/       # A post added here will be in the "course" category
+    machine-learning/
+        _posts/       # Same thing for the "machine-learning" category
+    ...               # There are other categories, you can add one too.
+    ~~~
+3.  Create the file `YYYY-MM-DD-title-of-your-review.markdown` in right folder (see above).
```
Comment (jodoin) [inline index.md]: Create a file 'YYYY-MM-DD-title-of-your-review.markdown` and put it in the right folder (see above).

File: index.md [hunk 5]
```diff
@@ -1,6 +1,61 @@
...
+    ~~~
+    _posts/           # A post added here will have no category
+    course/
+        _posts/       # A post added here will be in the "course" category
+    machine-learning/
+        _posts/       # Same thing for the "machine-learning" category
+    ...               # There are other categories, you can add one too.
+    ~~~
+3.  Create the file `YYYY-MM-DD-title-of-your-review.markdown` in right folder (see above).  
+**You must use this format : date at the beginning and no spaces.** Else the page won't be built. A valid name would be : `2017-01-31-going-deeper-with-convolutions.markdown`.
```
Comment (jodoin) [inline index.md]: You must use this format : a date at the beginning without any space.... Here is an example of a valid name :

File: index.md [hunk 6]
```diff
@@ -1,6 +1,61 @@
...
+    tags:   convolutional-networks deep-learning  # put relevant tags here
+    ---
+    
+    Content of your review.
+    ```
+    You can [preview your post while you write it](#previewing-your-post-locally) ; see the next section about this.
+5.  **Make a new branch**, commit your file and push.
+6.  [**Create a pull request**](https://github.com/vitalab/vitalab.github.io/compare) on the repo's github page.
+7.  **Add reviewers**: everybody that you think are knowledgeable about the subject or simply would be interested in your review.
+8.  When everybody is satisfied, **merge your branch and delete it**.
```
Comment (jodoin) [inline index.md]: When every reviewer has approved your branch, you can **merge it and delete it**.

File: index.md [hunk 7]
```diff
@@ -1,6 +1,61 @@
...
+
+This site is built around [**Jekyll**](https://jekyllrb.com/). Jekyll takes all the markdown files and generates a static html website.
+
+1.  [Install Ruby using rbenv](/how-to-install-ruby). Don't use `apt-get` since its version of Ruby is too old.
+2.  Install Jekyll by running : `gem install bundler jekyll`.
+3.  Go where you cloned the repo and run : `bundle install`. This will install the dependencies for our Jekyll site.
+4.  Run a local webserver using : `bundle exec jekyll serve`.
+5.  Access the site locally at : <http://127.0.0.1:4000/>
+
+Note that the site is automatically rebuilt when change a file.
```
Comment (jodoin) [inline index.md]: when a file is changed.

File: _posts/2017-01-27-welcome-to-vital-literature-review.markdown [hunk 8]
```diff
@@ -1,64 +0,0 @@
...
-title:     Welcome to the Literature Review site of VITAL
-date:      2017-01-27 12:18:56 -0500
-permalink: /welcome-how-to/
----
-
-<style>
-    li {
-        margin-bottom: 0.7rem; /* This will make the lists easier to read. */
-    }
-</style>
```
Comment (jodoin) [inline _posts/2017-01-27-welcome-to-vital-literature-review.markdown]: dd

File: about.md [hunk 9]
```diff
@@ -6,10 +6,4 @@ permalink: /about/
 
 Welcome to the Literature Review site of the [Videos & Images Theory and Analytics Laboratory (VITAL) of Sherbrooke University](http://vital.dinf.usherbrooke.ca/), headed by Professor Pierre-Marc Jodoin. This lab focuses on researching state of the art algorithms in image processing, computer vision, video analytics, medical imaging and statistical models. If you’d be interested in collaborating with us, do not hesitate to [contact us](http://vital.dinf.usherbrooke.ca/contact/)!
```
Comment (jodoin) [inline about.md]: Welcome to the Literature Review site of the [Videos & Images Theory and Analytics Laboratory (VITAL) of Sherbrooke University](http://vital.dinf.usherbrooke.ca/), headed by Professor Pierre-Marc Jodoin(http://info.usherbrooke.ca/pmjodoin).  Our lab specializes on  image processing, computer vision, video analytics, medical imaging and statistical modeling. If you’d be interested in collaborating with us, do not hesitate to [contact us](http://vital.dinf.usherbrooke.ca/contact/)!

File: _posts/2017-01-27-welcome-to-vital-literature-review.markdown (hunk 8 above)
Comment (lemairecarl) [inline _posts/2017-01-27-welcome-to-vital-literature-review.markdown]: dd?

**Pull request merged**